<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KLFKatePluginConfigWidget</class>
 <widget class="QWidget" name="KLFKatePluginConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,1,0,1,0,0" columnstretch="1,2,0,0">
   <item row="1" column="0" colspan="4">
    <widget class="QCheckBox" name="chkOnlyLatexMode">
     <property name="text">
      <string>Enable Previews only in latex highlighting mode</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Latex Preamble</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QPlainTextEdit" name="txtPreamble">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>path to KLatexFormula executable</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="KUrlRequester" name="pathKLF">
        <property name="mode">
         <set>KFile::ExistingOnly|KFile::File|KFile::LocalOnly</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QCheckBox" name="chkAutoPopup">
     <property name="text">
      <string>Pop-Up Automatically</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Popup Transparency:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSlider" name="sldTransparency">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>20</number>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="KIntSpinBox" name="spnTransparency">
     <property name="suffix">
      <string notr="true">%</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>20</number>
     </property>
     <property name="base">
      <number>10</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>sldTransparency</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spnTransparency</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spnTransparency</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sldTransparency</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
